mantid-developer-win.yml

```diff
@@ -23,7 +23,6 @@ dependencies:
   - matplotlib=3.6.*
   - nexus=4.4.*
   - ninja>=1.10.2
-  - numpy<=1.23
```

Comment:
I think it's probably best that we align our version of numpy for development with the one we're shippiung with, so pinning it to 1.20 would be appropriate

Reply:
I can't pin it to 1.20 (i.e. `numpy=1.20`) because it breaks the conda environment. 

```Could not solve for environment specs
Encountered problems while solving:
  - package euphonic-0.6.0-py36h68aa20f_0 requires python_abi 3.6.* *_cp36m, but none of the providers can be installed
```

If I use `numpy>=1.20` that is fine because it enables us to download the version that works best which seems to be `1.24.2` at the moment.